### PR TITLE
Basic front end

### PR DESCRIPTION
**Pull request by PerAsperaAdAstra3**
Basic front end elements and functionality finished. Some work remains to be done on Training.

### CODE REVIEW

File: WebContent/training.html [hunk 1]
```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
```
Comment (renatav) [inline WebContent/training.html]: Is this file needed?

File: src/main/java/training/controller/MainController.java [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.RequestMapping;
+
+@Controller
+public class MainController {
+
+	
+    @RequestMapping("/")
+    public String index(Model model) {
+        model.addAttribute("pageTitle", "Welcome to Training management application!");
```
Comment (renatav) [inline src/main/java/training/controller/MainController.java]: Is this controller used? If that is the case, translate the page title

File: src/main/resources/static/client.html [hunk 3]
```diff
@@ -1,10 +1,44 @@
...
-<html>
-<head>
-<meta charset="ISO-8859-1">
-<title>Insert title here</title>
-</head>
-<body>
-
-</body>
-</html>
+<html xmlns:th="http://www.thymeleaf.org">
```
Comment (renatav) [inline src/main/resources/static/client.html]: Is this file needed?

File: src/main/resources/templates/addClient.html [hunk 4]
```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
```
Comment (renatav) [inline src/main/resources/templates/addClient.html]: Not needed?

File: src/main/resources/templates/addExercise.html [hunk 5]
```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
```
Comment (renatav) [inline src/main/resources/templates/addExercise.html]: Not needed?

File: src/main/resources/templates/message.html [hunk 6]
```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html xmlns:th="http://www.thymeleaf.org">
```
Comment (renatav) [inline src/main/resources/templates/message.html]: Not needed

File: src/main/resources/templates/training.html [hunk 7]
```diff
@@ -0,0 +1,58 @@
...
+	<tbody>
+		<tr	id="tr-entity-list" th:each = "training : ${trainings}">
+			<td scope="row" class="clientId" th:utext="${training.date}"></td>
+			<td class="trainingNumberOfTra" th:utext="${training.numberOfTrainings}"></td>
+			<td class="trainingClientName" th:utext="${training.client}"></td>
+		</tr>
+	</tbody>
+</table>
+</form>
+<a th:href="@{/addTrainingToClient}"><button>Create training</button></a>
```
Comment (renatav) [inline src/main/resources/templates/training.html]: This should be translated and styled, but let's do that after merging

File: src/main/resources/templates/trainingCreation.html [hunk 8]
```diff
@@ -0,0 +1,323 @@
...
+		highlightRow(this)
+	});
+	
+	$(document).on("click", "#round-id-sync", function(event) {
+		selectRoundIdOnRowClick(this)
+	});
+	
+	  /*<![CDATA[*/
+    
+	function myFunctionName() {
```
Comment (renatav) [inline src/main/resources/templates/trainingCreation.html]: Name this function

File: src/main/resources/templates/trainingCreation.html [hunk 9]
```diff
@@ -0,0 +1,323 @@
...
+	      if (td.innerHTML.toUpperCase().indexOf(filter) > -1) {
+	        tr[i].style.display = "";
+	      } else {
+	        tr[i].style.display = "none";
+	      }
+	    }       
+	  }
+	}
+	  
+	function myFunctionDescription() {
```
Comment (renatav) [inline src/main/resources/templates/trainingCreation.html]: Change name

File: src/main/resources/templates/addClient.html (hunk 4 above)
Comment (PerAsperaAdAstra3) [reply src/main/resources/templates/addClient.html]: Removed.

File: src/main/resources/templates/addExercise.html (hunk 5 above)
Comment (PerAsperaAdAstra3) [reply src/main/resources/templates/addExercise.html]: Removed.

File: src/main/resources/static/client.html (hunk 3 above)
Comment (PerAsperaAdAstra3) [reply src/main/resources/static/client.html]: All unnecessary html files from static removed.

File: WebContent/training.html (hunk 1 above)
Comment (PerAsperaAdAstra3) [reply WebContent/training.html]: Removed.

File: src/main/java/training/controller/MainController.java (hunk 2 above)
Comment (PerAsperaAdAstra3) [reply src/main/java/training/controller/MainController.java]: Yes. It will be translated.

File: src/main/resources/templates/message.html (hunk 6 above)
Comment (PerAsperaAdAstra3) [reply src/main/resources/templates/message.html]: Removed.

File: src/main/resources/templates/training.html (hunk 7 above)
Comment (PerAsperaAdAstra3) [reply src/main/resources/templates/training.html]: Ok

File: src/main/resources/templates/trainingCreation.html (hunk 9 above)
Comment (PerAsperaAdAstra3) [reply src/main/resources/templates/trainingCreation.html]: Name changed to filterByDescription

File: src/main/resources/templates/trainingCreation.html (hunk 8 above)
Comment (PerAsperaAdAstra3) [reply src/main/resources/templates/trainingCreation.html]: Name changed to filterByName
